app/assets/javascripts/admin/addon/adapters/web-hook-event.js

```diff
@@ -4,4 +4,21 @@ export default class WebHookEvent extends RestAdapter {
   basePath() {
     return "/admin/api/";
   }
+
+  appendQueryParams(path, findArgs, extension) {
+    const urlSearchParams = new URLSearchParams();
+
+    for (const [key, value] of Object.entries(findArgs)) {
+      if (value && key !== "webhookId") {
+        urlSearchParams.set(key, value);
+      }
+    }
+
+    const queryString = urlSearchParams.toString();
+
+    if (queryString) {
```

Comment:
I think to make it a littttttttle prettier you could change 
```js
    if (queryString) {
      return `${path}/${findArgs.webhookId}${extension || ""}?${queryString}`;
    }
    return `${path}/${findArgs.webhookId}${extension || ""}`;
```
to
```js
let url =  `${path}/${findArgs.webhookId}${extension || ""};
if (queryString) {
  url = `${url}?${queryString}`
}
return url
```
just to not repeat that bit complex interpolation